diff --git a/component/tls/reality.go b/component/tls/reality.go
index ff6a0a3d..e1a5fe8b 100644
--- a/component/tls/reality.go
+++ b/component/tls/reality.go
@@ -71,9 +71,9 @@ func GetRealityConn(ctx context.Context, conn net.Conn, fingerprint UClientHello
 		binary.BigEndian.PutUint64(hello.SessionId, uint64(ntp.Now().Unix()))
 
 		copy(hello.SessionId[8:], realityConfig.ShortID[:])
-		hello.SessionId[0] = 1
-		hello.SessionId[1] = 8
-		hello.SessionId[2] = 2
+		hello.SessionId[0] = 26
+		hello.SessionId[1] = 7
+		hello.SessionId[2] = 11
 
 		//log.Debugln("REALITY hello.sessionId[:16]: %v", hello.SessionId[:16])
 
